base/utils/gin.go

```diff
@@ -27,6 +27,15 @@ func LoadParamInt(c *gin.Context, param string, defaultValue int, query bool) (i
 	return value, nil
 }
 
+func LoadParamBool(c *gin.Context, param string, defaultValue bool) bool {
```

Comment:
Please remove this function, since its not used anywhere.

Reply:
done